plugins/active_directory_ldap/komand_active_directory_ldap/actions/query_group_membership/action.py

```diff
@@ -21,11 +21,16 @@ def run(self, params={}):
         include_groups = params.get(Input.INCLUDE_GROUPS)
         expand_nested_groups = params.get(Input.EXPAND_NESTED_GROUPS)
         try:
-            group_dn = (
-                self.search_data(base=base, filter_query=f"(sAMAccountName={params.get(Input.GROUP_NAME)})")
-                .get("entries")[0]
-                .get("dn")
+            group = self.search_data(base=base, filter_query=f"(sAMAccountName={params.get(Input.GROUP_NAME)})").get(
+                "entries"
             )
+            if group and isinstance(group, list):
```

Comment:
```suggestion
            if group and isinstance(group, list) and len(group) > 0:
```

Reply:
This additional condition is not necessary. The `if group` expression handles cases where `group` is returned as an empty list.

Reply:
Confirmed
```
>>> l = []
>>> type(l)
<class 'list'>
>>> if l and isinstance(l, list):
...     print('true')
... else:
...     print('false')
...
...
false
```